composer.json

```diff
@@ -18,8 +18,8 @@
     ],
     "require": {
         "doctrine/mongodb-odm-bundle": "~3.0",
-        "doctrine/mongodb-odm": "~1.0",
-        "sonata-project/admin-bundle": "~2.3"
+        "sonata-project/admin-bundle": "~2.3.0",
```

Comment:
Well this will fail with symfony3, I would expect that master branch will support symfony3. 
Also this template change should break symfony <2.6